src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -43,6 +43,10 @@ public TagName getTagName() {
         return this.tagName;
     }
 
+    public List<TagTask> getModifiableTagTask() {
+        return this.tagTasks;
+    }
+
     public List<TagTask> getTagTasks() {
         return Collections.unmodifiableList(this.tagTasks);
     }
```

Comment:
The intention was for Tag to be immutable. (modifying fields is not allowed)
I'm not sure how `EditTagDescriptor` works though, so i'll leave it to your discretion.

Reply:
yep i removed it thanks!!